uvm/hier/usr/share/untangle/conf/log4j.xml

```diff
@@ -10,372 +8,82 @@
 
 <!-- $Id: log4j.xml,v 1.00 2012/04/12 21:57:25 dmorris Exp $ -->
 
```

Comment:
Better to add an comment stating that the configuration is also been updated programmatically from UvmContext

Reply:
Added a comment indicating `configuration is also been updated programmatically from UvmContextSelector`